include/odp/api/ipsec.h

```diff
@@ -10,22 +10,14 @@
  * ODP IPSEC API - platform specific header
  */
 
-#ifndef ODP_PLAT_IPSEC_H_
-#define ODP_PLAT_IPSEC_H_
+#ifndef ODP_API_IPSEC_H_
+#define ODP_API_IPSEC_H_
```

Comment:
it will be good to name defines without ODP_ to keep this naming only for API defines. But we can do that clean up later.

Reply:
Hmm. They are part of `<odp/api/FOO.h> space, so is define name.